packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/index.js

```diff
@@ -257,6 +266,28 @@ class Project extends React.Component {
             </a>
           </PropertyValue>
         </Item>
+        {sandbox.owned ? (
+          <Item flex>
+            <PropertyName>
+              Frozen
+              <Tooltip title="When true this sandbox will fork on edit">
+                <Icon />
+              </Tooltip>
+            </PropertyName>
+            <PropertyValue>
+              <FreezeConatainer>
+                <Switch
```

Comment:
Could you add the `small` prop here? I think it fits a bit better in the UI there 😄.

Reply:
Le switch is now small 🎉
<img width="252" alt="screen shot 2018-09-25 at 14 53 30" src="https://user-images.githubusercontent.com/1051509/46015536-d8eee480-c0d2-11e8-9116-1700525e4012.png">